### ISSUE TITLE
-include_libs is not correctly interpreted

### ISSUE THREAD

**Issue by elbrujohalcon** [AdRoll/rebar3_typer#33]
## `rebar3_typer` version
`0.0.1`

## Steps to reproduce
Create an umbrella app with two apps and these files:
* `apps/app1/src/my_mod.erl`
```erlang
-module(my_mod).

-include_lib("app2/include/inc.hrl").

-export([my_fun/0]).

my_fun() -> ?BANANAS.
```
* `apps/app2/include/inc.hrl`
```erlang
-define(BANANAS, bananas).
```

## Current behavior
If you run `rebar3 typer` on the root folder, you get…
```bash
/tmp/umbrella$ rebar3 typer
===> Analyzing applications...
===> Compiling rebar3_typer
===> Looking for types to add...
===> typer: Analysis failed with error report:
apps/app1/src/my_mod.erl:3:14: can't find include lib "app2/include/inc.hrl"
apps/app1/src/my_mod.erl:7:14: undefined macro 'BANANAS'

apps/app1/src/my_mod.erl:5:2: function my_fun/0 undefined
```
Which is the same thing you'll get if you run typer this way…
```bash
/tmp/umbrella$ typer --plt _build/default/rebar3_24.0.2_plt apps/app1/src/my_mod.erl
typer: Analysis failed with error report:
apps/app1/src/my_mod.erl:3:14: can't find include lib "app2/include/inc.hrl"
apps/app1/src/my_mod.erl:7:14: undefined macro 'BANANAS'

apps/app1/src/my_mod.erl:5:2: function my_fun/0 undefined
```

## Expected behavior
```bash
/tmp/umbrella$ rebar3 typer
===> Analyzing applications...
===> Compiling rebar3_typer
===> Looking for types to add...
===>
%% File: "apps/app1/src/my_mod.erl"
===> %% --------------------------------
===> -spec my_fun() -> 'bananas'.
```
…which is what you get if you run TypEr like this…
```bash
/tmp/umbrella$ typer --plt _build/default/rebar3_24.0.2_plt -pa _build/default/lib/app2/ebin apps/app1/src/my_mod.erl

%% File: "apps/app1/src/my_mod.erl"
%% --------------------------------
-spec my_fun() -> 'bananas'.
```

## TL;DR
We need to add the paths for all apps (including dependencies) recognized by `rebar3` to the code path before running `typer_core`.
